scripts/collect-assets.js

```diff
@@ -14,7 +14,7 @@ const cli = parseCli(
   {
     org: "konveyor",
     repo: "kai",
-    releaseTag: "v0.1.0",
+    releaseTag: "v0.2.0-pre.1",
```

Comment:
can we update rulesets tag to [0.7.1](https://github.com/konveyor/rulesets/releases/tag/v0.7.1)?